src/http.ts

```diff
@@ -2770,7 +2770,12 @@ export class LemmyHttp extends Controller {
     }
 
     if (!response.ok) {
-      throw new Error(json["error"] ?? response.statusText);
+      let err: Error = {
+        name: json.error ?? response.statusText,
```

Comment:
Is the `statusText` supposed to be the HTTP status code?

Reply:
The [error string associated with that code.](https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText). So `404` would return `Not Found`.

We've never i18n'd those type errors, but maybe we can at some point.